ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/config/LoopFlowExtensionParameters.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.search_tree_rao.config;
+
+import com.farao_community.farao.rao_api.RaoParameters;
+import com.powsybl.commons.extensions.AbstractExtension;
+
+/**
+ * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ */
+public class LoopFlowExtensionParameters  extends AbstractExtension<RaoParameters> {
+
+    static final boolean DEFAULT_RAO_WITH_LOOP_FLOW = false; //loop flow is for CORE D2CC, default value set to false
```

Comment:
I don't understand why we let the possibility to have a LoopFlowExtensionParameters with raoWithLoopFlow = false. If we want to monitor loopflows in the RAO, we will extend the RaoParameters with a LoopFlowExtensionParameters. If we don't want that, we won't. Am I wrong?

Reply:
As commented by Baptiste, I put it directly in SearchTreeRaoParameters. OK?

Reply:
yes it's perfect !